### PR TITLE
Feature/image slider

### PR DESCRIPTION
**Pull request by Leon910**

### CODE REVIEW

File: components/CreateEmotionTypeForm.js [hunk 1]
```diff
@@ -47,62 +48,66 @@ export default function CreateEmotionTypeForm({ onSubmit }) {
...
-
-        <StyledButtonContainer>
-          <StyledLinkCancel href="/">Cancel</StyledLinkCancel>
-          <StyledButtonSubmit type="submit">Submit</StyledButtonSubmit>
-        </StyledButtonContainer>
-        {formError && <StyledError>{formError}</StyledError>}
-      </StyledEmotionForm>
-    </StyledFormContainer>
+    <>
+      <SwiperSlider />
```
Comment (F-Kirchhoff) [inline components/CreateEmotionTypeForm.js]: this component is now inside the Form component, which is a bit confusing. If I would use the "CreateEmotionTypeForm" I would not expect to have a SwiperSlider as well. You can Move this Component to the page where you also use the Form and place it in front of it: 

```js
<SwiperSlider />
<CreateEmotionTypeForm />
```

File: components/SwiperSlider.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        slidesPerView={1}
+        freeMode={true}
+        pagination={{ clickable: true, type: "bullets" }}
+        autoplay={{
+          delay: 5000,
+          disableOnInteraction: false,
+        }}
+      >
+        {images.map((src, index) => (
+          <SwiperSlide key={index}>
```
Comment (F-Kirchhoff) [inline components/SwiperSlider.js]: never use the index of an array item as the key. either use the image url itself or define an Id for each image element.

**Review by F-Kirchhoff — CHANGES_REQUESTED** [review]
Very nice, I added 2 hints what to improve, but all in all it looks great. And nice that you credited the photographers.

File: components/CreateEmotionTypeForm.js (hunk 1 above)
Comment (Leon910) [reply components/CreateEmotionTypeForm.js]: Great! I put the component inside the parent component createemotiontype.

File: components/SwiperSlider.js (hunk 2 above)
Comment (Leon910) [reply components/SwiperSlider.js]: ohh another learning! Thanks again for reminding me @F-Kirchhoff. I added an id and access it in the mapping!

File: components/SwiperSlider.js [hunk 3]
```diff
@@ -9,9 +9,9 @@ import styled from "styled-components";
 
 export default function SwiperSlider() {
   const images = [
-    "/images/wave1.jpg",
-    "/images/wave2.jpg",
-    "/images/wave3.jpg",
+    { id: 1, src: "/images/Slider-1.svg" },
+    { id: 2, src: "/images/Slider-2.svg" },
+    { id: 3, src: "/images/Slider-3.svg" },
   ];
```
Comment (F-Kirchhoff) [inline components/SwiperSlider.js]: ✨

File: components/SwiperSlider.js [hunk 4]
```diff
@@ -26,12 +26,12 @@ export default function SwiperSlider() {
...
-        {images.map((src, index) => (
-          <SwiperSlide key={index}>
+        {images.map((image) => (
+          <SwiperSlide key={image.id}>
             <ImageWrapper>
               <Image
-                src={src}
-                alt={`Test image ${index + 1}`}
+                src={image.src}
+                alt={`${image.id}`}
```
Comment (F-Kirchhoff) [inline components/SwiperSlider.js]:
```suggestion
                alt={image.id}
```

**Review by F-Kirchhoff — APPROVED** [review]

**Pull request merged**